index.js

```diff
@@ -3,7 +3,7 @@ import execa from 'execa';
 
 // Reference: https://www.gaijin.at/en/lstwinver.php
 const names = new Map([
-	['11.0', '11'],
+	['10.0.22', '11'],
```

Comment:
Is it guaranteed that Windows 11 will always be `10.0.11`?

Reply:
Really I don't know.. I've found nothing clear about next Windows 11 release versions.. Preview releases keep the `10.0.22` pattern. But I think that this will be OK.

Reply:
@sindresorhus From my latest reads, it seems that this will be OK as it. Windows 11 should keep releases like `10.0.22xxx`.

But in all cases, this is better than the actual code which does not recognize Windows 11 at all.

Reply:
This line will need a code comment that explains why it's `10.0.22`, that it's not going to change, and also link to a source for the info..

Reply:
Sorry, but as I said before, I've not found any "official" documentation about future Windows 11 releases information.. All I've found is https://docs.microsoft.com/en-us/windows/release-health/windows11-release-information and Wikipedia. But I've read in different places (forums, etc.) that Windows 11 will keep a `10.0.22xxx` format for it's releases.. If needed, and that this pattern changes, we can improve this in the future !

But for now, I think that this will be enough. The actual version of `os-name` recognize Windows 11 as Windows 10 and this PR fix this. If later Windows 11 use a release `10.0.23xxx`, this fix will not work anymore, but we can react when (and if) this will be the case !

If we wait some months, I think that we could find a "pattern" in the releases version of Windows 11.. And if required, improve the logic of this package. But for now, `os-name` should recognize Windows 11 and this fix do the tricks.

What do you think about all that ?

Reply:
> All I've found is docs.microsoft.com/en-us/windows/release-health/windows11-release-information and Wikipedia. But I've read in different places (forums, etc.) that Windows 11 will keep a 10.0.22xxx format for it's releases.. 

I'm ok with unofficial sources. But the code really needs a comment explaining why it's done like it is. Because the code looks totally wrong if you don't have the context.

Reply:
Yes, I understand your need. I've add the officiel Microsoft link about Windows 11 release history to the code. I think that this is enough. Because we does not explain why Windows 8.1 is `6.3` for exemple.. The only validation for this point is that we have all the Windows 8.1 releases history and that we could find this pattern. Which is not the case with Windows 11 now because only one version was released !

As I said before, if we wait 1-2 months, your link to https://www.gaijin.at/en/lstwinver.php will perhaps be updated with the Windows 11 informations and then this will be OK.

Sorry, but I cannot do more. Hop this is OK for you !